### PR TITLE
[BugFix] fix alter lake table "in_memory" property bug

### PR DESCRIPTION
**Pull request by abc982627271**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #
When alter lake table's in_memory property, it reports this error:
image

Given the fact that 'in_memory' property will be deprecated in the future, we will do nothing but only return ok when the modified table is a lake table.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [x] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 1 / 2 (50.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/alter/SchemaChangeHandler.java|1|2|50.00%|[1289]|

File: fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java [hunk 1]
```diff
@@ -1283,6 +1284,10 @@ public void updatePartitionTabletMeta(Database db,
         db.readLock();
         try {
             OlapTable olapTable = (OlapTable) db.getTable(tableName);
+            // LakeTable not support update tablet meta
+            if (olapTable instanceof LakeTable) {
+                return;
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java:1289]: I think it is better to throw unsupported exception.

**Review by gengjun-git — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 4 / 6 (66.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/alter/Alter.java|4|6|66.67%|[495, 515]|

**Review by wyb — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Comment by wyb** [comment]
run starrocks_admit_test

**Pull request merged**